Pod/Classes/SEGMoEngageInitializer.m

```diff
@@ -10,49 +10,49 @@
 #import <MoEngageSDK/MoEngageSDK.h>
 
 
-static MOSDKConfig* currentSDKConfig = nil;
+static MoEngageSDKConfig* currentSDKConfig = nil;
 static NSString * const segmentVersion = @"7.1.0";
```

Comment:
@deepa-iOS-SDK  update the segment version

Reply:
Done